### ISSUE TITLE
Thanks for the new stuff!

### ISSUE THREAD

**Issue by Vitae666** [Wiedmolol/CQAutomater#3]
Obviously not an "Issue" but GitHub doesn't believe in just "Comments" :)
What are the chances that you will be implementing Experiments with diff timers included? Compared to the stuff that you are adding this is minor on the "want to have" things.

As a question, I got from a chest "Got 50 Disasters(not rewarded in game)"
Does it mean I didn't get it or is "Not rewarded' another way of saying "Not a hero"?

**Comment by Wiedmolol** [comment]
I can't add experiments as they are not handled by server. 

Disasters and Energy Boost rewards are lost. When you open a chest in game the server sends reward to the game. UM, Heroes and Followers are kept on server so your game client doesn't need to do anything to "claim" them but disasters and energy are handled by game, not by server. If you open a chest with my app the reward obtained is sent to the app, not to the game so game doesn't know that you rolled disasters/energy hence the reward is lost.

**Comment by Vitae666** [comment]
Understood on both counts. Weird how the experiments aren't server side. As well 😁

**Comment by Vitae666** [comment]
Just to confirm tho, we still do get the actual reward (heroes/disasters/energy) in game and it isn't actually lost, right?

**Comment by saynotoid** [comment]
as stated in the warnin! in the app 'disasters or production boosts will be lost' - but who cares anyway lol:)

keep up the GREAT work man!!! autoclaimer gets more and more OP

edit: the only thing that bothers me so far is app crash after waking up:
im playing/doin stuff with an app running in the background
then im fallin asleep and my pc does the same in about 15-30 minutes and the app doesnt work which is ok ofc,
but when i woke up in the morning it throws an login failed dialog then help with macrocreating then crash window - i have to focus and close them all one by one then restart the app - thats a bit annoying

**Comment by Vitae666** [comment]
Hrmm..musta missed the warning 😁
Kinda useful depending on what dimension you are due to the value. But yeah, it's a trade off. Either manual or pay a "cost" LOL

**Comment by Vitae666** [comment]
Silly question: When doing the DQ's I get 2 Normal and 1 Hero chest. Will this not open them until the Countdown runs down? So that it will do 3 Normal and 1 Hero or will it only run for the 1 Normal from the countdown?
Asking because it hasn't collected yet and I've restarted the prog just in case that was needed. Guess I'll find out in a bit under an hour, either way :)

EDIT:
Ok, nope, won't collect the other chests. Just the 1.

**Comment by saynotoid** [comment]
here is an issue: ive started an autoclaimer its said ive got 2-2 normal heroes chests, also i had my macrocreator runnin and w8in, then
Got 20 Disasters(not rewarded in game)
Got 50 Disasters(not rewarded in game)
Got Ailen
Got Ailen
i 'got' these
then ive 'get heroes' in macro creator a few times - still lvl 2 ailen
then ive logged to the cosmos in my browser and still - only lvl 2 ailen, so

where the hell did my reward go? thats very frustrating so far:(

**Comment by Vitae666** [comment]
That's weird. Mine never collected the extras. At this point I'm glad 😂

**Comment by Wiedmolol** [comment]
Sorry, I've used wrong hero name list so you got one of the heroes but it's not Ailen. If I'm reading it right you might got Alpha instead. I'm gonna upload new version really soon with corrected hero order so it tells you correct rewards.

EDIT: New version out.

**Comment by saynotoid** [comment]
thx 4 a quick response.

**Comment by saynotoid** [comment]
would be awesome to have a 'refresh' btn on the DQ tab if u knowhat i mean

**Comment by Vitae666** [comment]
I'd mentioned on Kong that there's times where the DQ timer in game and prog go out of synch. It had happened days before that update where Lucy was added and the timer was reset. 
I'd  made a suggestion of an hourly or so auto-refresh.

I've also noticed plenty of times where from day to day there's a wild diff when the DQ's are do able. One day it was when I was waking up at 6am, the next it was all the way at 4PM, a few days later it was at 9PM and days later 1PM. Don't know what causes the times to shift because it sure as hell isn't because the DQ's aren't on a 24hr cycle. There's no accounting for hours going up or down in the span of a few days the way it has.
